src/app/table/data-table/terra-data-table.component.ts

```diff
@@ -33,6 +33,7 @@ export class TerraDataTableComponent<S extends TerraBaseService, D extends Terra
     @Input() inputService:S;
     @Input() inputDataType:string;
     @Input() inputHasCheckboxes:boolean;
+    @Input() inputHasPager:boolean;
```

Comment:
kann man das outputDoPaging event dann trotzdem ausführen? XD

Reply:
nein, das wird für den Anwendungsfall in der Contact UI anders gelöst